spec/lib/activitypub/tag_manager_spec.rb

```diff
@@ -170,11 +170,6 @@
       expect(subject.uri_to_resource(subject.uri_for(status), Status)).to eq status
     end
 
-    it 'returns the local status for OStatus tag: URI' do
-      status = Fabricate(:status)
-      expect(subject.uri_to_resource(OStatus::TagManager.instance.uri_for(status), Status)).to eq status
-    end
```

Comment:
Note that the test was actually not testing for this, as `OStatus::TagManager.instance.uri_for(status)` was returning `status.uri`, and not the `tag:` URI.